ironfish/src/rpc/routes/chain/broadcastTransaction.test.ts

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { useTxSpendsFixture } from '../../../testUtilities/fixtures'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+
+describe('Route wallet/broadcastTransaction', () => {
+  const routeTest = createRouteTest()
+
+  it('should broadcast a transaction', async () => {
+    const { account, transaction } = await useTxSpendsFixture(routeTest.node, {
```

Comment:
You can use a simpler faster fixture. You aren't relying on the spends here so you don't need a TX with spends. Why not use a simple miners fee transaction?